docs/src/dataDisplay/table/index.jsx

```diff
@@ -1,210 +1,443 @@
+                    </Typography>
 
-                {/* Font Size */}
-                <TableSampleFontSize />
+                    <Typography
+                        variant="body2"
+                    >
+                        <span className="font-weight-semibold">Note:</span>
+                        &nbsp;
+                        Must have
+                        <code>fixed</code>
+                        enabled.
+                    </Typography>
+                </MarkdownContainer>
 
-                {/* Full Width Table */}
-                <TableSampleFullWidth />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleSingleLine').default}
+                >
+                    <TableSampleSingleLine />
+                </Example>
 
-                {/* Selectable Table */}
-                <TableSampleSelectable />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Size
+                    </Heading>
 
-                {/* Single Line Table */}
-                <TableSampleSingleLine />
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s default cell size (vertical gutters) can be changed.
+                    </Typography>
+                </MarkdownContainer>
 
-                {/* Size */}
-                <TableSampleSize />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleSize').default}
+                >
+                    <TableSampleSize />
+                </Example>
 
-                {/* Stretch Table */}
-                <TableSampleStretch />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Stretch Table
+                    </Heading>
+                </MarkdownContainer>
 
-                {/* Striped Table */}
-                <TableSampleStriped />
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleStretch').default}
+                >
+                    <TableSampleStretch />
+                </Example>
 
-                {/* Active */}
-                <TableSampleActive />
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Active
+                    </Heading>
 
-                {/* Disabled */}
-                <TableSampleDisabled />
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s row or cell can be active.
```

Comment:
**NAB**
It also seems to me that 'row' and 'cell' should be plural here, since a table is likely to have many of them.

Reply:
Fixed.

Reply:
Looks good.  Resolving.